src/main/java/clover/util/Parser.java

```diff
@@ -0,0 +1,70 @@
+package clover.util;
+
+import clover.command.*;
+
+public class Parser {
+    public static Command parse(String input) {
+        String[] split = input.split(" ", 2);
+        String command = split[0].toLowerCase();
+
+        switch (command) {
```

Comment:
Perhaps you can put the switch block into a try catch block and catch the respective exceptions so that u don't need to repeat the try catch block